js/index.js

```diff
@@ -1 +1,110 @@
 // Your code goes here
+
+//Nav Bar effects/////////////////////////////////////////////////////
+const nav = document.querySelectorAll('.nav-link');
+
+nav[0].addEventListener('click', e => {
+    nav[0].style.color = '#ff0000';
+    console.log('This is my event listener');
+});
+
+nav[1].addEventListener('click', e => {
+    nav[1].style.color = '#0000ff';
+    e.preventDefault();               //Prevent page reload <====================
+});
+
+nav[2].addEventListener('click', e => {
+    nav[2].style.color = '#008000';
+});
+
+nav[3].addEventListener('click', e => {
+    nav[3].style.color = '#ffa500';
+});
+
+//h1 logo effects///////////////////////////////////////////////////
+const logo = document.querySelector('.logo-heading');
+
+logo.addEventListener('mouseover', e => {
+    logo.style.backgroundColor = '#ffff00';
+})
+
+logo.addEventListener('mouseout', e => {
+    logo.style.backgroundColor = '#ffffff';
+})
+
+//h2 'Welcome to  the Fun bus'//////////////////////////////////////
+
+const welcome = document.querySelector('header h2');
+
+welcome.addEventListener('mouseover', e => {
+    welcome.style.fontSize = '4rem';
+    console.log('test');
+});
+
+
+//h4////////////////////////////////////////////////////////////////////not working
+
+const h4 = document.querySelectorAll('h4');
+
+h4.forEach(item => {item.addEventListener('mouseover', e => {
+    h4.style.fontSize = '4rem';
```

Comment:
Your trying to apply the style of the node list here you want to apply it each individual h4 
```suggestion
   item.style.fontSize = '4rem';
```